tests/testthat/testEdgeThresh.R

```diff
@@ -0,0 +1,251 @@
+# edgeThresh.R
+#
+# Structure your tests according to the three principles expressed below:
+#    test_that("parameter errors are correctly handled", { ... })
+#    test_that("a sane input gives an expected output", { ... })
+#    test_that("a corrupt input does not lead to corrupted output", { ...})
+
+
+context("Graph permutation and delta calculation")
+
+
+# ==== BEGIN SETUP AND PREPARE =================================================
+# OLOG <- as.character(getOption("rete.logfile"))   # save original logfile name
+# logFileName(fPath = tempdir(), setOption = TRUE)  # make tempdir() the log dir
+# NL <- .PlatformLineBreak()
+# if (file.exists(logName)) { file.remove(logName)}
+
+# ((Is rete.EGGprototype only the barebones valid EGG, or does it contain testable
```

Comment:
The prototypes ultimately should contain all features that .checkArgs() would need to perform a validity check.